angularjs/angular.d.ts

```diff
@@ -708,7 +708,6 @@ declare module angular {
     // see http://docs.angularjs.org/api/ng.$window
     ///////////////////////////////////////////////////////////////////////////
     interface IWindowService extends Window {
-        [key: string]: any;
```

Comment:
I know this was the intention and I understand if it won't be accepted because it's a breaking change. However, it's not nice to do this from the d.ts. There must be a reason why the Microsoft guys didn't do this already in lib.d.ts, in the Window interface, right?. If someone wants to override this, he/she can merge with the Window interface and extend it, although it's not really a good practice. It shouldn't be encouraged as default behavior.